Recruit-Training/Embedded-Recruit-Training/Core/Inc/BlueLedToggleTask.h

```diff
@@ -6,7 +6,12 @@
 #include "cmsis_os.h"
 
 //TODO: Define an external CAN handle (same variable as in main)
+CAN_HandleTypeDef hcan2;
+
 //TODO: Define external CAN_Tx header (same variable as in main)
+CAN_TxHeaderTypeDef CAN_TxHeader;
+
 //TODO: Define external blue LED toggle flag (same variable as in main)
+uint8_t blueToggleFlag;
```

Comment:
The above variables should have the `extern` keyword. That's what we meant by external. `extern` extends the visibility of these variables, which will mean that instead of creating new variables, the linker will use the ones you defined in main.